docs/adr/adr-009-telemetry.md

```diff
@@ -0,0 +1,226 @@
+# ADR #009: Telemetry
+
+## Changelog
+
+* 2022-07-4: Started
+* 2022-07-10: Initial Draft finished
+* 2022-07-11: Stylistic improvements
+
+## Authors 
+
+@Wondertan @liamsi
+
+## Legend
+- "ShrEx" - Share Exchange
+> It's all ogre now
+
+## Context
+> Now I know why I don't like writing ADRs - because I cannot run/test them and see if they work or not.
+> Hoping that quality team feedback will solve this problem!
+
+Celestia Node needs deeper observability of each module and their components. The only integrated observability solution 
+we have is logging and there are two more options we need to explore from the observability triangle(tracing, metrics and logs).
+
+There are several priorities and "why"s we need deeper observability:
+* Analysis of the current p2p share exchange or "ShrEx" stack
+  * So we can evaluate real world Full Node reconstruction qualities, along with data availability sampling
+  * And adjust our roadmap accordingly.
+* Incentivized Testnet
+  * Tracking participants and validation that do task correctly
+  * So all participants provide to us valuable data/insight/traces that we can analyze and improve on
+* Monitoring dashboars
+  * For Celestia's own DA network infrastructure, e.g. DA Network Bootstrappers
+  * For the node operators
+* Extend debugging arsenal for the networking heavy DA layer
+  * Local development
+  * Issues found with Testground
+  * Production
+* Establishing metrics/data driven engineering culture for celestia-node devs
+  * Metrics and tracing allows extracting dry facts out of any software on its performance, liveness, bottlenecks,
+    regressions, etc., on whole system scale, so devs can reliably respond
+  * Basing on these, all the improvements can be proven with data _before_ and _after_ a change
```

Comment:
I think that should be the first bullet point on top.

Reply:
Changed